aries-site/src/pages/foundation/icons.mdx

```diff
@@ -44,85 +53,89 @@ For guidance on installing and migrating to use the @hpe-design/icons-grommet ic
   secondary
 />
 
-### Accessibility
-
-For screen reader accessibility there is an aria-label provided for
-each icon. The color of the icons should also pass any color
-contrast ratio.
-
-### Icon within components
-
-Icons can be used in a variety of different ways. They are used within other components such as [Search](/components/search), [Menu](/components/menu), [Button](/components/button).
-
-<Example
-  docs="https://icons.grommet.io/?theme=hpe"
-  code="https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/foundation/icons/IconComponentExample.js"
-  figma="https://www.figma.com/design/Hio1n5z95fdxSxuS86dLqJ/HPE-Icons---V2?node-id=66-13&t=S5gUdNzCkbPmEoZF-1"
->
-  <IconComponentExample />
-</Example>
-
 ### Icon sizes
 
-The default icon size is medium. This should be adjusted depending
-on where in your application the icon is being used.
+The default icon size is medium. Icons can be scaled using predefined size options to align with our typography scale, ensuring consistent vertical rhythm and proper inline alignment with text.
 
 <Example code="https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/foundation/icons/IconSizeExample.js">
   <IconSizeExample />
 </Example>
 
-### Button icons
+### Icon color
 
-The icon should be centered aligned with any text that is being used next to it. The correct size icon should be paired with similar size text. When using the Icon as a button both the label and icon will be wrapped within that Button.
+Standalone icon components use the default color token **icon-default**.\
+Decorative icons may also use **icon-strong**, **icon-weak**, **icon-disabled**, or **icon-primary** to convey emphasis or state.\
```

Comment:
We should align on a content style guide related to how we represent values, either inline code markdown or bold, but not both. 

```suggestion
Decorative icons may also use `icon-strong`, `icon-weak`, `icon-disabled`, or `icon-primary` to convey emphasis or state.
```

cc: @julauxen